### DISCUSSION TITLE
load balancing active/active cluster

### DISCUSSION THREAD

**Discussion by shay-xie** [nextgenhealthcare/connect#5636] [General]
Hello,

we have two mirth nodes building up, using the same shared SQL database. 
and will use the  advanced cluster plug-in 
and would like to implement the load balance with  Citrix ADC / Netscaler

anyone has done that already ?  I have searched the nextgen forum and google, could not get an answer.
any input is greatly appreciated

**Comment by shay-xie** [comment]
my thoughts is, maybe this avanvanced cluster plug-in is good at ative/passive  cluster, but what about the active /active cluster ?

**Comment by shay-xie** [comment]
https://forums.mirthproject.io/forum/mirth-connect/general-discussion/182974-load-balancing-active-active-cluster

I also had it posted in the "official" Nextgen forum

**Comment by jonbartels** [comment]
Thank you for posting the forum link. That helps other users find answers! 

The Advanced Clustering User Guide is available online. It doesn't discuss active/active that I can find though :( 
 https://docs.nextgen.com/bundle/Connect_Advanced_Clustering_User_Guide/page/connect/connect/topics/c_Advanced_Clustering_introduction.html
